test/integration/targets/ecs_cluster/playbooks/full_test.yml

```diff
@@ -0,0 +1,5 @@
+- hosts: localhost
+  connection: local
+
+  roles:
+    - ecs_cluster
```

Comment:
This isn't something you changed, but do you mind changing this line to 'yes'? https://github.com/ansible/ansible/blob/devel/test/integration/targets/ecs_cluster/tasks/main.yml#L20
(if you don't have the correct instance role already the tests don't work because of that)

Reply:
ah right, thanks